frontend/app/src/composables/message-handling/calendar-reminder.ts

```diff
@@ -0,0 +1,57 @@
+import dayjs from 'dayjs';
+import { type Notification, Severity } from '@rotki/common';
+import type { CalendarEventPayload } from '@/types/history/calendar';
+
+interface UseCalendarReminderHandler {
+  handle: (data: CalendarEventPayload) => Notification;
```

Comment:
Do you think we can standarize this a bit by extracting a generic interface? since all the handlers follow a similar format?

Reply:
yeah I should think of it, updated now!